admin_guide/cns_recommendations.adoc

```diff
@@ -0,0 +1,52 @@
+### CNS - Container Native Storage recommendations for OCP
+
+OCP platform offers CNS storage feature which makes easier for OCP users to fulfill their
+storage needs. With CNS solution users and admins are empowered to have together storage and
+application pods running together on same infrastructure and sharing same resources,this leads
+to better resource utilization.
+Follow CNS https://access.redhat.com/documentation/en-us/red_hat_gluster_storage/3.1/html-single/container-native_storage_for_openshift_container_platform_3.4/[Documentation]
+for instructions how to configure CNS as part of OCP cluster.
+
+Per CNS documentation currently it is supported up to 100 CNS volumes
+on one Trusted Storage Pool ( TSP )
+
+==== CNS volumes create time
+When deploying applications on OCP it is important to estimate how long
+it will take to build environment from storage side in order for an application pod to able to start.
+If application pod requires an PVC (Persistent Volume Claim) then at below graph
+is presented how long it takes for different number of CNS to be created and bound to corresponding PVC.
+All tests were performed on one TSP (Trusted Storage Pool) and using hardware configuration for
+CNS per CNS https://access.redhat.com/documentation/en-us/red_hat_gluster_storage/3.1/html-single/container-native_storage_for_openshift_container_platform_3.4/[Documentation]
+
+Dynamic storage provisioning and storage classes were aslo configured and used when provisioning PVC
+
+.Time to create different number of CNS volumes
+image::https://raw.githubusercontent.com/ekuric/openshift/master/cns/images/create_10_50_100.png[Create CNS volumes]
+
+***Theoretically it is possible to create more than 100 CNS volumes in one
+trusted storage pool, but currently it is not supported***
+
+==== CNS volumes delete time
+
+When user deletes PVC claim which is used by an application pod, then that action
+will trigger deletion of CNS volume which was used by PVC.
+Persistent Volume Claims will disappear immediately from ***oc get pvc*** view, however the the time to delete
+and recycle CNS volumes will depend on number of CNS volumes
+
+Below graphs shows these dependencies
+
+.Time to delete different number of CNS volumes
+image::https://raw.githubusercontent.com/ekuric/openshift/master/cns/images/delete_10_50_100.png[Delete CNS volumes]
```

Comment:
same comment as above, can this be phrased as "deletion time of CNS volumes scales linearly up to 100 volumes"...